### ISSUE TITLE
is there a way to refresh the channel list page forcefully?

### ISSUE THREAD

**Issue by abhirup-patra** [GetStream/stream-chat-android#1513]
Is there a way to refresh the page forcefully after programmatically deleting some channels?

**Comment by JcMinarro** [comment]
Hello @abhirup-patra 
Could you explain what do you mean and what you want to achieve?
Could you share with us what you do?
Whenever you delete a channel, you should receive an event from our WebSocket connections that notify you about that. Isn't it enough for your use-case?

**Comment by abhirup-patra** [comment]
actually the thing that i am doing is just checking for blank channel when the channel list loads and upon finding blank channel i am programmatically deleting that. But it is only reflected when i restart the application. Not in real time. Is there any way to do so?

**Comment by abhirup-patra** [comment]
```
if(channelState.getMessages().size()<0)
{

                        channelState.setHidden(true);
                        client = ChatClient.instance();
                        ChannelClient channelClient = client.channel("messaging",channelState.getId());

                        channelClient.delete().enqueue(result -> {

//                            loadChannels.loadChannels();
//                            if (result.isSuccess()) {
//                                Channel channel = result.data();
//                                channel.setHidden(true);
//
//
//                            }
                            return null;
                        });
                    }
```

this is how i am doing it programmatically. But it doesnt get reflected immediately. Is there any refresh function available?

**Comment by rupinderjeet** [comment]
> actually the thing that i am doing is just checking for blank channel when the channel list loads and upon finding blank channel i am programmatically deleting that. But it is only reflected when i restart the application. Not in real time. Is there any way to do so?

I think this is similar to #1473

**Comment by JcMinarro** [comment]
It was already fixed on #1499 and is released on the last version
